docs/gitops/README.md

```diff
@@ -40,6 +40,19 @@ You can use the Janus IDP Demo repository to install the `Red Hat OpenShift Pipe
 
 The OpenShift Pipelines Operator can be installed directly from the OperatorHub. Select the operator from the list and install it without any special configuration.
 
+Make sure you install an OpenShift Pipelines Operator's version compatible with your Orchestrator Operator's version:
+
+| OpenShift Pipelines Operator version          |  Orchestrator Operator Version |
+|-----------------------------------------------|--------------------------------|
+| 4.11                                          |1.3.x+                          |
```

Comment:
perhaps the suffix `+` isn't required, if that could be interpreted to 1.4

Reply:
Yeah, I was not sure about it either